helm/README.md

```diff
@@ -0,0 +1,35 @@
+# Helm-based deployment
+
+The Wire platform is deployed on top of Kubernetes. This certainly includes all *stateless* services (e.g. Brig, Galley),
+but may or may not include *stateful* backing services (e.g. Cassandra or Elasticsearch).
+
+The respective Kubernetes objects are defined in Helm charts. This allows to template and transfer them. 
+The charts themselves are defined in [wire-server](https://github.com/wireapp/wire-server/tree/master/charts)
+and uploaded two the [release](https://s3-eu-west-1.amazonaws.com/public.wire.com/charts) or
+[develop](https://s3-eu-west-1.amazonaws.com/public.wire.com/charts-develop) Helm repositories.
+
+To describe a deployment in a declarative fashion a tool called [*Helmfile*](https://github.com/roboll/helmfile) is
+being used, which wraps the `helm` CLI. 
+
+
+## Deploy environment created by `terraform/environment`
+
+An 'environment' is supposed to represent all the setup required for the Wire
+platform to function.
+
+'Deploying' an environment means creating the respective Objects on top of Kubernetes
+to instantiate all the services that together represent the Wire backend.
+playbooks against servers specified in an inventory, resulting in a fully This action
+can be re-run as often as you want (e.g. in case you change some variables or upgrade
+to new versions).
```

Comment:
There is some problem here with these sentences, looks like a copy paste snippet of text at the wrong place.

Reply:
Great catch. Fixed.